src/channel/httpBeaconChannel.ts

```diff
@@ -48,8 +49,9 @@ export class HttpBeaconChannel implements DuplexChannel<string, Envelope<string,
         return fetch(endpointUrl, {
             method: 'POST',
             headers: {
-                'X-Application-Id': appId,
+                'X-App-Id': appId,
                 'X-Client-Id': await cidAssigner.assignCid(),
+                'X-Client-Library': CLIENT_LIBRARY,
```

Comment:
@denis-rossati, could you please update the affected tests?

Reply:
Sorry for my delay, it's done. The new changes doesn't seem to add new branches to cover